README.md

```diff
@@ -31,3 +31,8 @@ See also http://dotmatrixtool.com
 ## Contributing
 
 Contributions are welcome. Submit pull requests to the `usbserial` branch.
+
+When preparing your pull request, please be sure:
+
+1. the tool still works with connected LED Matrix Input Modules
```

Comment:
Nit: the -> The

Reply:
I prefer it being lower case since they're not proper sentences